index.d.ts

```diff
@@ -1837,7 +1833,14 @@ declare namespace Eris {
     unbanGuildMember(guildID: string, userID: string, reason?: string): Promise<void>;
     unpinMessage(channelID: string, messageID: string): Promise<void>;
     validateDiscoverySearchTerm(term: string): Promise<{ valid: boolean }>;
-    on: ClientEvents<this>;
+    emit<K extends keyof ClientEvents>(event: K, ...args: ClientEvents[K]): boolean;
+    emit(event: string, ...args: any[]): boolean;
```

Comment:
Don't think we need `emit` to be typed as the only time it'll be used is within Eris internals

Reply:
I personally use it to reemit interactions for regular command handling (causes less duplicated code, and I don't feel like reinitializing my event listeners)

And what can something like this really hurt? It still allows for the addition of arbitrary events via the last line

Reply:
I feel like for a normal use case, including this seems pretty pointless

Reply:
Eh seems ok